WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/UseCases/RefundFeesCalculationUseCase.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import Yosemite
+
+struct RefundFeesCalculationUseCase {
```

Comment:
Some documentation on this type won't hurt! 😇

Reply:
True! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/5663/commits/f891a9a68b905b0c40bd8bade3c113cf35a75d98. Let me know if this documentation sounds good or should be improved!